spec/support/service_spec_helper.rb

```diff
@@ -8,4 +8,12 @@ def default_headers
       'x-rh-insights-request-id' => 'gobbledygook',
       'original_url'             => 'some_url' }
   end
+
+  def original_url
+    "whatever"
```

Comment:
👍 Nice

Reply:
@eclarizio We might not need these changes anymore since they were incorporated in this PR #https://github.com/ManageIQ/catalog-api/pull/218

Reply:
@mkanoor If I remove the `:original_url` key from the `#default_request` method, I get spec errors. Are we sure that the `'original_url' => 'some_url'` is in the right place?

Originally I was using this defined `original_url` but now the only place that is using it is the `#default_request` method, so it could just be hardcoded there instead.